src/v2/BaseLoginRouter.js

```diff
@@ -130,6 +130,26 @@ export default Router.extend({
     }
   },
 
+  handleDeviceEnrollmentFlow () {
+    const deviceEnrollmentValue = this.settings.options.deviceEnrollment;
+    const rawIdxStateForDeviceEnrollment = {
+      stateHandle: this.settings.get('stateToken'),
+      version: '1.0.0',
+      expiresAt: '2020-06-14T22:15:50.000Z',
+      intent: 'LOGIN',
+      deviceEnrollment: {
+        type: 'object',
+        value: deviceEnrollmentValue
+      }
+    };
+
+    this.handleIdxResponseSuccess({
```

Comment:
create a new event  `terminalStateWithNoRemediation ` and listen to it at line 82
and handler `handleTerminalStateWithNoRemediation`